pkg/alertmanager/amcfg.go

```diff
@@ -1327,6 +1340,51 @@ func (cb *configBuilder) convertMSTeamsConfig(
 	return out, nil
 }
 
+func (cb *configBuilder) convertJiraConfig(ctx context.Context, in monitoringv1alpha1.JiraConfig, crKey types.NamespacedName) (*jiraConfig, error) {
+	out := &jiraConfig{
+		VSendResolved:     in.SendResolved,
+		Project:           in.Project,
+		Labels:            in.Labels,
+		Summary:           in.Summary,
+		Description:       in.Description,
+		Priority:          in.Priority,
+		IssueType:         in.IssueType,
+		ResolveTransition: in.ResolveTransition,
+		ReopenTransition:  in.ReopenTransition,
+		WontFixResolution: in.WontFixResolution,
+	}
+
+	if in.APIURL != nil {
+		out.APIURL = in.APIURL
+	} else if cb.cfg.Global != nil && cb.cfg.Global.JiraAPIURL != nil {
+		out.APIURL = ptr.To(cb.cfg.Global.JiraAPIURL.RequestURI())
+	}
+
+	if len(in.Fields) > 0 {
+		outFields := make(map[string]interface{})
```

Comment:
```suggestion
		outFields := make(map[string]interface{}, len(in.Fields))
```

Since we know up front the map size is a good idea initialise with the proper size.

Reply:
Thanks @nicolastakashi

Reply:
I see it's resolved but change not applied, is it intentional @dongjiang1989 ?

Reply:
Hmmm... Added ref: https://github.com/prometheus-operator/prometheus-operator/pull/7285/commits/5d8dbc3e107238618a116b5ab900c7f6865a08af